app/src/main/res/layout/saved_sightings_activity.xml

```diff
@@ -20,15 +20,31 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recViewLocations"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
+        android:layout_width="197dp"
```

Comment:
You shouldn't need specific/hardcoded dimensions, if you are using constraint layout.  I'd leave this as it were.